src/style.css

```diff
@@ -0,0 +1,72 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url("../src/fonts/Avenir-Book.ttf");
+}
+
+html {
+  font-family: Avenir, sans-serif;
+}
+
+.search_box {
+  position: relative;
+  margin-top: 20px;
+}
+
+.search_icon {
+  position: absolute;
+  width: 19px;
+  height: 19px;
+  top: 25px;
+  bottom: 26px;
+  left: 26px;
+  right: 855px;
+}
+
+.search {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
```

Comment:
```suggestion
  font-weight: 900;
```

Reply:
when you typing, the text should be bold, font-weight: 300  - that's for placeholder